docker/bootstrap/Dockerfile.mysql57

```diff
@@ -16,8 +16,8 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --no-ins
         echo percona-server-server-5.7 percona-server-server/root_password password 'unused'; \
         echo percona-server-server-5.7 percona-server-server/root_password_again password 'unused'; \
     } | debconf-set-selections && \
-    percona-release enable-only tools \
-    apt-get update -y && \
+    apt-get update -y && apt-get install -y percona-release && \
```

Comment:
Can't we delete the bootstrap versions of mysql57 and percona57? We don't support those versions any more (since v19).

Reply:
#16620 has been opened for this